src/Compiler/FSComp.txt

```diff
@@ -1593,6 +1593,7 @@ featurePreferExtensionMethodOverPlainProperty,"prefer extension method over plai
 featureWarningIndexedPropertiesGetSetSameType,"Indexed properties getter and setter must have the same type"
 featureChkTailCallAttrOnNonRec,"Raises warnings if the 'TailCall' attribute is used on non-recursive functions."
 featureUnionIsPropertiesVisible,"Union case test properties"
+featureBoolPartialActivePattern,"bool representation for partial active patterns"
```

Comment:
Since it also allows omitting the `[<return: Struct>]`, I would word it differently, something like "Boolean and type-directed partial active patterns". Because, now if you will omit `[<return: Struct>]` and return `'a voption` in <=8, it compiler will tell you about "bool representation", which is a bit misleading.

Reply:
Thanks for the more proper new name for this feature!

Reply:
The feature was renamed to "Boolean-returning and return-type-directed partial active patterns", this might be a more proper name for it.